app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="container-fluid d-flex flex-column justify-content-center auth_container shadow-sm bg-body-tertiary rounded">
+  <h2>Sign up</h2>
+
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field field_input">
+      <br />
+      <%= f.text_field :name, autofocus: true, placeholder:"Name", autocomplete: "name" %>
+    </div>
+
+    <div class="field field_input">
+      <br />
+      <%= f.email_field :email, autofocus: true, placeholder:"Email", autocomplete: "email" %>
+    </div>
+
+    <div class="field field_input">
+      <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %><br />
+      <%= f.password_field :password, placeholder:"Password", autocomplete: "new-password" %>
+    </div>
+
+    <div class="field field_input">
+      <br />
+      <%= f.password_field :password_confirmation, placeholder:"Password confirmation", autocomplete: "new-password" %>
+    </div>
+    <br>
+    <div class="actions submit">
+      <%= f.submit "Sign up" %>
+    </div>
+  <% end %>
+  <%= render "devise/shared/links" %>
+</div>
```

Comment:
- After signing up, the user gets redirected to the LogIn page, and then, after filling out the form, users are stuck on still on the same page, instead of navigating to the app. Please, ensure users can access the rest of your webpage after they sign in or sign up. 
- Also, kindly work on the height of the page so that users can see any message at the top of the page, right now, it's very difficult to see any message directing them what to do, as this is not a good user experience. Please, kindly improve this.

Reply:
After signing up you have to confirm your account by clicking the link sent to you via email from there you can sign in and be redirected to the app. 
All messages appear above the heading and is visible actually. You could try creating the same account to see where the error message like below
![image](https://github.com/SFarida/rails_blog_application/assets/23447217/b90e828c-78ff-486a-8b6f-b87e6c012a50)